src/features/books/books.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = { value: [] };
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.value.push(action.payload);
+    },
+
+    removeBook: (state, action) => {
+      const st = state;
+      st.value = st.filter((state) => state.id !== action.payload);
+    },
+  },
+});
+
+export const { addBook, removeBook } = booksSlice.actions;
+
+export default booksSlice.reducer;
```

Comment:
- [ ] Good job @anagudelogu , however, The value of each action type needs to follow the [ducks pattern]so it will be something like:  ```const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK``` kindly fix this